parcels/field.py

```diff
@@ -350,9 +368,15 @@ def reshape(self, data, transpose=False):
             logger.warning_once("Casting field data to np.float32")
             data = data.astype(np.float32)
         if transpose:
-            data = np.transpose(data)
+            if isinstance(data, np.ndarray):
+                data = np.transpose(data)
```

Comment:
can we use the same pythonic writing as further down here and below: `lib = np if isinstance(data, np.ndarray) else da`?